eng/Versions.props

```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
   <!-- Versioning for assemblies/packages -->
   <PropertyGroup>
-    <MajorVersion>7</MajorVersion>
+    <MajorVersion>8</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
```

Comment:
Do we need/want to start setting this to a specific preview version?

Reply:
What would the number correspond to?

Reply:
Presumably, the NET preview version.

Reply:
So it would be like `rc.2` now? Who would be maintaining the number?

Also I don't think there's exact correspondence anyway - Razor inserts the same version into both [8 RC 1](https://github.com/dotnet/sdk/pull/35260) and [9 alpha 1](https://github.com/dotnet/sdk/pull/35235) it seems. Looking elsewhere for inspiration, aspnetcore does what you suggest, but they seem to be inserting different branch into 8 RC 1 vs 9 alpha 1. Roslyn, on the other hand, inserts the same build like Razor, and it uses VS preview version for the label here, but its minor version also corresponds to VS, so that's different.

Reply:
If we're fine with not updating it, that's ok with me. Was just throwing it out there as a suggestion.